javatests/arcs/core/storage/StorageProxyTest.kt

```diff
@@ -185,25 +185,26 @@ class StorageProxyTest {
             suspend {
                 // handle reads data
                 storageProxy.getParticleView()
-                // make sure we follow up with a sync so the particle view doesn't get block
+                // make sure we follow up by ensuring view is synced so the particle view doesn't
+                // stay blocked
                 launch {
                     delay(10)
                     val op = mock(CrdtOperation::class.java)
                     mockCrdtModel.appliesOpAs(op, true)
-                    storageProxy.applyOp(op)
+                    storageProxy.onMessage(ProxyMessage.Operations(listOf(op), null))
                 }
             }
         )
 
         val workers = 20
         val jobs = 10
         Executors.newFixedThreadPool(workers).asCoroutineDispatcher().use {
-            val newScope = it // does it@ work for this?
+            val inPool = it
```

Comment:
You can just name the lambda argument `inPool`, right?

Reply:
derp. Yes that would be much better, thank you.